test/unit_tests/unittest_tenth_of_degree.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2020 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#include <gtest/gtest.h>
+
+#include "psen_scan_v2/angle_conversions.h"
+#include "psen_scan_v2/tenth_of_degree.h"
+
+using namespace psen_scan_v2;
+
+namespace psen_scan_v2_test
+{
+TEST(TenthOfDegreeTest, constructorTest)
+{
+  TenthOfDegree tenth_of_degree{ 1 };
+}
+
+TEST(TenthOfDegreeTest, valueTest)
+{
+  TenthOfDegree tenth_of_degree{ 1 };
+  EXPECT_EQ(tenth_of_degree.value(), 1);
+}
+
+TEST(TenthOfDegreeTest, toRad)
+{
+  TenthOfDegree tenth_of_degree{ 1 };
+  EXPECT_EQ(tenth_of_degree.toRad(), tenthDegreeToRad(1));
```

Comment:
```suggestion
  EXPECT_DOUBLE_EQ(tenth_of_degree.toRad(), tenthDegreeToRad(1));
```
or did I miss something?